docker/addons/influxdb-writer/docker-compose.yml

```diff
@@ -54,7 +54,7 @@ services:
     networks:
       - docker_mainflux-base-net
     volumes:
-      - ./subjects.toml:/config/subjects.toml
+      - ./config.toml:/config.toml
```

Comment:
is this path ok?

Reply:
Yes, because the path is changed from `/config` to the root (`/`).